zulipterminal/config/keys.py

```diff
@@ -323,77 +323,77 @@ class KeyBinding(TypedDict):
     'BEGINNING_OF_LINE': {
         'keys': ['ctrl a'],
         'help_text': 'Jump to the beginning of line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'END_OF_LINE': {
         'keys': ['ctrl e'],
         'help_text': 'Jump to the end of line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'ONE_WORD_BACKWARD': {
         'keys': ['meta b'],
         'help_text': 'Jump backward one word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'ONE_WORD_FORWARD': {
         'keys': ['meta f'],
         'help_text': 'Jump forward one word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'DELETE_LAST_CHARACTER': {
         'keys': ['ctrl h'],
         'help_text': 'Delete previous character (to left)',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'TRANSPOSE_CHARACTERS': {
         'keys': ['ctrl t'],
         'help_text': 'Transpose characters',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_END_OF_LINE': {
         'keys': ['ctrl k'],
         'help_text': 'Cut forwards to the end of the line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_START_OF_LINE': {
         'keys': ['ctrl u'],
         'help_text': 'Cut backwards to the start of the line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_END_OF_WORD': {
         'keys': ['meta d'],
         'help_text': 'Cut forwards to the end of the current word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'CUT_TO_START_OF_WORD': {
         'keys': ['ctrl w'],
         'help_text': 'Cut backwards to the start of the current word',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'PASTE_LAST_CUT': {
         'keys': ['ctrl y'],
         'help_text': 'Paste last cut section',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'UNDO_LAST_ACTION': {
         'keys': ['ctrl _'],
         'help_text': 'Undo last action',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'PREV_LINE': {
         'keys': ['up', 'ctrl p'],
         'help_text': 'Jump to the previous line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
     'NEXT_LINE': {
         'keys': ['down', 'ctrl n'],
         'help_text': 'Jump to the next line',
-        'key_category': 'msg_compose',
+        'key_category': 'editor',
     },
```

Comment:
These appear to be applicable in all the cases, which I was a little surprised by - perhaps at some point I assumed that some text boxes didn't wrap, or that we had to pass an explicit multiline parameter.

I remember occasional strange behavior with using these keys in the past, and now these are the widgets used for the other input boxes it looks like they may be a problem there too. This is a separate issue to discuss or file, but I'd suggest exploring the behavior with long text - we or urwid-readline may have a bug there.

Reply:
I posted about this in `#zulip-terminal>Up/Down bug (urwid-readline?)`, to ensure we keep track of that.